paths/graphql/04-query-info.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Grab a Query's Response
+permalink: /graphql/query-info
+next-page: /graphql/another-query
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  ![img](../images/gifs/graphql/recent-prs.gif)
+
+  1. Using the response from the query in the last step, fill in this template.
```

Comment:
I would ❤️ to have a javascript box within the page where they copy the output of their query into a box, and it puts the format for the template in another box. We could also grab info at that point so we'd know how many times it was being used. Maybe in the next iteration?